test_scripts/Polices/build_options/039_ATF_P_TC_PTU_HMI_Level_Affected_Apps_FULL_LIMITED_HTTP.lua

```diff
@@ -111,97 +116,105 @@ function Test:TestStep_RegisterFirstApp()
       EXPECT_RESPONSE(correlationId, { success = true })
       EXPECT_NOTIFICATION("OnPermissionsChange")
     end)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2)
 end
 
 function Test:TestStep_ActivateAppInFull()
   commonSteps:ActivateAppInSpecificLevel(self,HMIAppID,"FULL")
+  self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "FULL", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+  :Do(function(_,data) print("SDL -> MOB1: OnHMIStatus, level: " .. data.payload.hmiLevel) end)
 end
 
 function Test:TestStep_UpdatePolicyAfterAddFirstAp_ExpectOnHMIStatusNotCall()
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
-  :Do(function()
-      local CorIdSystemRequest1 = self.mobileSession:SendRPC("SystemRequest",
-        {
-          requestType = "HTTP",
-          fileName = "ptu1app.json",
-        },"files/ptu1app.json")
-      self.mobileSession:ExpectResponse(CorIdSystemRequest1, {success = true, resultCode = "SUCCESS"})
+
+  local CorIdSystemRequest1 = self.mobileSession:SendRPC("SystemRequest",
+    {
+      requestType = "HTTP",
+      fileName = "ptu1app.json",
+    },"files/ptu1app.json")
+  self.mobileSession:ExpectResponse(CorIdSystemRequest1, {success = true, resultCode = "SUCCESS"})
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UP_TO_DATE"})
+
+  self.mobileSession:ExpectNotification("OnPermissionsChange")
+  -- Expect after updating HMI status will not change
+  self.mobileSession:ExpectNotification("OnHMIStatus"):Times(0)
+  commonTestCases:DelayedExp(10000)
+end
+
+function Test:TestStep_RegisterSecondApp()
+  local onsystemreq_app1 = false
+  local onsystemreq_app2 = false
+  self.mobileSession1 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession1:StartService(7)
+  :Do(function (_,_)
+      local correlationId = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered")
+      :Do(function(_,data)
+          HMIAppID = data.params.application.appID
+        end)
+      self.mobileSession1:ExpectResponse(correlationId, { success = true })
+      self.mobileSession1:ExpectNotification("OnPermissionsChange")
     end)
-  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate")
-  :ValidIf(function(exp,data)
-      if
-      (exp.occurences == 1 or exp.occurences == 2) and
-      data.params.status == "UP_TO_DATE" then
-        return true
+
+  self.mobileSession1:ExpectNotification("OnSystemRequest", {requestType = "HTTP"}):Times(Between(1,2)) --"LOCK_SCREEN_ICON_URL" + HTTP
```

Comment:
@istoimenova, Please remove expectation of "HTTP" only since it's expected to get 2 types